appium/tests/helpers/ElementHelper.ts

```diff
@@ -40,7 +40,7 @@ class ElementHelper {
   }
 
   static checkStaticText = async (element: WebdriverIO.Element, label: string) => {
-    await this.waitElementVisible(element);
+    await element.waitForDisplayed({timeout: DEFAULT_TIMEOUT});
```

Comment:
What is the difference between `await this.waitElementVisible(element);` and `await element.waitForDisplayed({timeout: DEFAULT_TIMEOUT});` ?

Reply:
await this.waitElementVisible(element); -  calls the method element.waitForDisplayed({timeout: DEFAULT_TIMEOUT})
I think better use element.waitForDisplayed({timeout: DEFAULT_TIMEOUT})
we call element.waitForDisplayed({timeout: DEFAULT_TIMEOUT}) directly, i think it works more stable and faster

Reply:
No such thing. The two are 100% equivalent.

Reply:
I think waitUntilVisible doesn't use default timeout, I ran tests locally and catch the error too, it was long spinner on message loading, we can try to check all email fields(subject, message body, sender) and after that we will check the badge text

Reply:
@tomholub now it is without retry
<img width="1297" alt="Screenshot 2021-12-08 at 13 57 06" src="https://user-images.githubusercontent.com/91205853/145204655-587f4867-16f5-44ed-9cac-88d042be3a45.png">

Reply:
There is no apparent difference between between the original code and the new one. It may be passing right now, but that doesn't mean that this change is meaningful.

Reply:
Does appium have a "repeat-until-failure" mode for running specs?

Reply:
im not sure about this, need to check

Reply:
@tomholub we can user this structure for checking element displayed
`    let count = 0;
    do {
      await browser.pause(1000);
      count++;
    } while (await (await this.enterPassPhraseField).isDisplayed() !== true && count <= 15);`